index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="project1.css">
+	<title>MyPortfolio</title>
+</head>
+<body>
+	<header>
+		
+		<nav class="navbar">
+			<a href="#about" value="About">About</a>
+			<a href="#portfolio" value="Portfolio">Portfolio</a>
+			<a href="#contact">Contact</a>
+			<a href="LiamLawrence.pdf" download>Download CV</a>
+		</nav>
+		
+	<section id="top-of-the-page" class="intro-section">
```

Comment:
this section should be `header` and not just a section. The `header` is generally the section that contains `h1`